__init__.py

```diff
@@ -19,8 +20,12 @@ def table_to_dict(tbl_name):
     return tbl.to_pandas().to_dict(orient='list')
 
 
-def create_empty_dict(key_names, arr_size=0):
-    dictionary = {}
+def create_empty_dict(key_names, arr_size=0, ordered=False):
+    if ordered:
+        dictionary = OrderedDict()
+    else:
+        dictionary = {}
+        
```

Comment:
There are some nice ways to initialize a `dict` with keys.  See:
https://stackoverflow.com/questions/2241891/how-to-initialize-a-dict-with-keys-from-a-list-and-empty-value-in-python

The dict-comprehension and the following option might work.

Reply:
Also, you might avoid using `OrderedDict`.  This was in place since Python 3.6, I believe: https://stackoverflow.com/questions/39980323/are-dictionaries-ordered-in-python-3-6